addons/misc/CfgWeapons.hpp

```diff
@@ -16,4 +16,73 @@ class CfgWeapons
             scope = 0;
         };
     };
+
+    class ItemCore;
+    class kat_Armband_Red_Cross_Item: ItemCore {
+
+        detectRange = -1;
+        author = "Miss Heda, vccv9040 (Swedish Forces Pack)";
+        simulation = "ItemMineDetector";
+        displayName = CSTRING(Armband_Red_Cross_ItemName);
+        scope = 2;
+        cost = 2;
+        side = 3;
+        model = QPATHTOF(models\armband\armband.p3d);
+        editorPreview = QPATHTOF(ui\ArmbandWhiteCross.paa);
+        picture = QPATHTOF(ui\ArmbandWhiteCross.paa);
+        class ItemInfo: InventoryItem_Base_F {
+            mass = 3;
+        };
+    };
+
+    class kat_Armband_Medic_Item: ItemCore {
```

Comment:
you could make all armband items inherit from 'kat_Armband_Red_Cross_Item' so you don't have to set thinks like 'cost, side and scope' everytime